doc/api/http2.md

```diff
@@ -1064,7 +1064,7 @@ const server = http2.createServer();
 server.on('stream', (stream) => {
   function statCheck(stat, headers) {
     // Check the stat here...
-    stream.respond({ ':status': 304 });
+    stream.respond({ ':status': 304 }, { endStream: true });
```

Comment:
Side note: I've been thinking about making methods like `respond()` chainable. There are a number of places where we could do it.